app/jobs/rdv_solidarites_webhooks/process_rdv_job.rb

```diff
@@ -73,8 +77,21 @@ def applicants
       @applicants ||= Applicant.where(rdv_solidarites_user_id: rdv_solidarites_user_ids)
     end
 
-    def applicant_ids
-      applicants.pluck(:id)
+    def participations
+      # TODO : Comment gérer le destroy d'une participation ?
+      # coté rdv-s cette action (d'admin) n'envoi pas de notifications aux usagers
+      @participations ||= \
+        rdv_solidarites_rdv.participations.map do |participation|
+          applicant = Applicant.find_by(rdv_solidarites_user_id: participation.user[:id])
```

Comment:
On pourrait ici itérer sur un array de `applicants` pour pas faire de requête supplémentaire :)